### ISSUE TITLE
[Warning] S.M.A.R.T. output reading error: exit status 1

### ISSUE THREAD

**Issue by denisgolius** [Sheridan/smartctl_exporter#2]
When I run it 
```
[Warning] S.M.A.R.T. output reading error: exit status 1
panic: runtime error: index out of range

goroutine 39 [running]:
main.(*SMARTctlInfo).mineVersion(0xc0001ceeb8)
	/mnt/Sheridan/sheridan/sheridan/smartctl_exporter/smartctlinfo.go:46 +0x92a
main.(*SMARTctlInfo).Collect(...)
	/mnt/Sheridan/sheridan/sheridan/smartctl_exporter/smartctlinfo.go:35
main.SMARTctlManagerCollector.Collect(0xc0001bc120)
	/mnt/Sheridan/sheridan/sheridan/smartctl_exporter/main.go:34 +0x1b0
github.com/prometheus/client_golang/prometheus.DescribeByCollect.func1(0x9688c0, 0xca8930, 0xc0001bc120)
	/mnt/Sheridan/sheridan/sheridan/smartctl_exporter/vendor/src/github.com/prometheus/client_golang/prometheus/collector.go:90 +0x3b
created by github.com/prometheus/client_golang/prometheus.DescribeByCollect
	/mnt/Sheridan/sheridan/sheridan/smartctl_exporter/vendor/src/github.com/prometheus/client_golang/prometheus/collector.go:89 +0x75
```

But if  I run **smartctl -i /dev/sda** , I get this normal output
```
smartctl 6.6 2016-05-31 r4324 [x86_64-linux-4.15.0-55-generic] (local build)
Copyright (C) 2002-16, Bruce Allen, Christian Franke, www.smartmontools.org
=== START OF INFORMATION SECTION ===
Vendor:               FTS
Product:              PRAID CP400i
Revision:             4.66
User Capacity:        6,000,606,183,424 bytes [6.00 TB]
Logical block size:   512 bytes
Physical block size:  4096 bytes
Logical Unit id:      0x6003005702999a9021e4fa780a81c811
Serial number:        0011c8810a78fae421909a9902570003
Device type:          disk
Local Time is:        Sat Aug 17 16:58:56 2019 EEST
SMART support is:     Unavailable - device lacks SMART capability.
```

![Screenshot_20190817_165705](https://user-images.githubusercontent.com/5650611/63212630-28d20200-c110-11e9-9586-dc471c31fb76.png)

**Comment by Sheridan** [comment]
Sorry for the long answer...
Can ypu send me result of `smartctl --json --xall /dev/sda > /tmp/sda.json`, json file?

**Comment by Sheridan** [comment]
oh, you have smartctl 6.6, but import to json [released in 7.0](https://www.smartmontools.org/browser/tags/RELEASE_7_0/smartmontools/NEWS#L11)
